tests/test_threads.py

```diff
@@ -132,3 +132,56 @@ class WX:
             pool = ThreadPool(window)
             pool.post_event(None)
             WX.PostEvent.assert_called_once()
+
+
+def test_run_in_thread():
+    with patch("pyguitemp.threads.ThreadPool", MagicMock()):
+        from pyguitemp.threads import ThreadPool, run_in_thread
+
+        def target():
+            pass
+
+        run_in_thread(target)
+        assert ThreadPool().run_thread.call_count == 1
+        ThreadPool().run_thread.assert_called_once_with(target, None, None, None, None)
+
+
+def test_run_daemon():
+    with patch("pyguitemp.threads.ThreadPool", MagicMock()):
+        from pyguitemp.threads import ThreadPool, run_daemon
+
+        run_daemon(lambda: None)
+        ThreadPool().run_daemon_called_once_with(None, None, None, None, True)
+
+
+def test_abort_thread():
```

Comment:
There should be two separate tests, one for `abort_thread` and another one for `should_abort`. If you find you need to repeat too much code, extract that as `fixtures`.